src/main/resources/openapi/audiorequests.yaml

```diff
@@ -289,6 +289,77 @@ paths:
                 title: "The requested data cannot be located"
                 status: 500
 
+  /audio-requests/playback:
+    get:
+      tags:
+        - Audio Requests
+      summary: Playback audio
+      operationId: playback
+      description: Playback audio
+      parameters:
+        - in: query
+          name: media_request_id
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: OK
+          content:
+            application/zip:
```

Comment:
Is this the right content for playback?
Should it be `audio/mpeg` or maybe something else for MP3 playback?